demo/laneLoader.js

```diff
@@ -528,143 +533,177 @@ function updateSegments(laneSegments, clonedBoxes, prevIsContains, point, index,
+
+    for (let s of supplierNum) {
+      for (let d of laneDirection) {
+        for (let n of laneNum) {
+          layerName = `Supplier${s}_${d}_Lane${n}`;
+          filename = `Supplier${s}_${d}_Lane${n}.fb`;
+          loadLanesHelper(layerName, filename, s);
+        }
+      }
+    }
+    layerName = `TomTom`;
+    filename = `I-75-North_potree.fb`;
+    loadLanesHelper(layerName, filename, 1);
+  }
+  // Load Comparison Dataset (hardcoded to one for now)
+  if (comparisonDatasets.length > 0) {
+    filename = 'lanes.fb';
+    tmpSupplierNum = -2;
+    await loadLanes(s3, bucket, comparisonDatasets[0], filename, tmpSupplierNum, window.annotateLanesModeActive, viewer.scene.volumes, (laneGeometries) => {
+      let lanesLayer = new THREE.Group();
+      lanesLayer.name = `Lanes-${comparisonDatasets[0].split("Data/")[1]}`;
+      addLaneGeometries(laneGeometries, lanesLayer);
+      viewer.scene.dispatchEvent({
+        "type": "truth_layer_added",
+        "truthLayer": lanesLayer
+      });
+    });
+  }
 } // end of loadLanesCallback
 
 // add an event listener for the reload lanes button
-export function addReloadLanesButton() {
-	window.annotateLanesModeActive = false; // starts off false
-
-	$("#reload_lanes_button")[0].style.display = "block"
-	let reloadLanesButton = $("#reload_lanes_button")[0];
-	reloadLanesButton.addEventListener("mousedown", () => {
-
-		let proceed = true;
-		if (window.annotateLanesModeActive) {
-			proceed = confirm("Proceed? Lanes will be reloaded, so ensure that annotations have been saved if you want to keep them.");
-		}
-
-		if (proceed) {
-			// REMOVE LANES
-			let removeLanes = viewer.scene.scene.getChildByName("Lanes");
-			while (removeLanes) {
-				viewer.scene.scene.remove(removeLanes);
-				removeLanes = viewer.scene.scene.getChildByName("Lanes");
-			}
-
-			// Pause animation:
-			animationEngine.stop();
-
-			// TOGGLE window.annotateLanesModeActive
-			window.annotateLanesModeActive = !window.annotateLanesModeActive;
-
-			// Disable Button:
-			reloadLanesButton.disabled = true;
-
-			{
-				$("#loading-bar")[0].style.display = "none";
-				setLoadingScreen();
-				loadLanesCallback(s3, bucket, name, () => {
-					removeLoadingScreen();
-
-					// TOGGLE BUTTON TEXT
-					if (window.annotateLanesModeActive) {
-						reload_lanes_button.innerText = "View Truth Lanes";
-						document.getElementById("download_lanes_button").style.display = "block";
-					} else {
-						reload_lanes_button.innerText = "Annotate Truth Lanes";
-						document.getElementById("download_lanes_button").style.display = "none";
-					}
-
-					reloadLanesButton.disabled = false
-
-				});
-			}
-		}
-	});
+export function addReloadLanesButton () {
+  window.annotateLanesModeActive = false; // starts off false
+  $("#reload_lanes_button")[0].style.display = "block"
+  let reloadLanesButton = $("#reload_lanes_button")[0];
+  reloadLanesButton.addEventListener("mousedown", () => {
+    let proceed = true;
```

Comment:
Don't use variables if you don't need them. Conditional expressions are clearer than `if` statement logic.
```
    const proceed = window.annotateLanesModeActive ?
                    confirm("Proceed? Lanes will be reloaded, so ensure that annotations have been saved if you want to keep them.") : 
                    true
```